### PR TITLE
DEB-100: Corrected the wrong format when opening in left.

### PR DESCRIPTION
**Pull request by PurwaShrivastava**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/qlik-date-picker
   docker:
-    - image: circleci/node:stretch
+    - image: circleci/node:11.15.0
```
Comment (AlbertBackenhof) [inline .circleci/config.yml]: You need to rebase against the qlik-date-picker branch. I've already fixed this problem by making the extension build with gulp 4.

File: src/qlik-date-picker.js [hunk 2]
```diff
@@ -71,6 +71,17 @@ define(["qlik", "jquery", "./lib/moment.min", "./calendar-settings", "css!./lib/
             html += '</div>';
             return html;
         }
+        function getPosition( element ) {
+            if (element.offset().left < 600) {
+                return "right";
+            }
+            else if (element.offset().right < 600) {
```
Comment (AlbertBackenhof) [inline src/qlik-date-picker.js]: the "else if" and "else" both end up with the same result, so then a simple "else" should be enough.

File: .circleci/config.yml (hunk 1 above)
Comment (PurwaShrivastava) [reply .circleci/config.yml]: Shall do that !

**Comment by PurwaShrivastava** [comment]
Shall do the changes in a new branch, closing this because of merge conflicts.

**Pull request closed without merging**